srcalc/src/main/webapp/WEB-INF/views/displayResults.jsp

```diff
@@ -23,24 +23,24 @@
     </c:forEach>
     </ol>
     <ol>
-    <li><button id="signCalculationButton" class="button-em" type="submit">Sign Calculation</button></li>
-    <li>
-    <%-- Add the required patientDfn parameter, preserving the patient from the current calculation. --%>
-    <c:url var="newCalcUrl" value="/newCalc"><c:param name="patientDfn" value="${calculation.patient.dfn}"/></c:url>
-    <a href="${newCalcUrl}" class="btn-default">Start New Calculation</a></li>
     <li>
         <c:url var="enterVarsUrl" value="/enterVars"/>
         <a href="${enterVarsUrl}" class="btn-default">Return to Variable Input Form</a>
     </li>
+    <li><button id="signCalculationButton" class="button-em" type="submit">Sign Calculation</button></li>
     </ol>
+    <%-- Add the required patientDfn parameter, preserving the patient from the current calculation. --%>
+    <c:url var="newCalcUrl" value="/newCalc"><c:param name="patientDfn" value="${calculation.patient.dfn}"/></c:url>
```

Comment:
The signature warning is now strangely out of place: how about putting it on top of the buttons?